examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase2.php

```diff
@@ -225,6 +228,7 @@ private static function createCampaign(
         );
 
         $campaignResourceName = $campaignResponse->getResults()[0]->getResourceName();
+        /** @var MutateCampaignsResponse $campaignResponse */
```

Comment:
You put this in a wrong place? Should be above before line 222.

Reply:
Whops, wrong cmd+v.
Fixed.